dist/AjaxDynamicDataTable.js

```diff
@@ -0,0 +1,90 @@
+function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import DynamicDataTable from "./DynamicDataTable";
+import { AxiosInstance as axios } from "axios";
+
+class AjaxDynamicDataTable extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      rows: [],
+      currentPage: 1,
+      totalPages: 1,
+      orderByField: null,
+      orderByDirection: null
+    };
+  }
+
+  componentDidMount() {
+    this.loadPage(1);
+  }
+
+  render() {
+    const {
+      rows,
+      currentPage,
+      totalPages,
+      orderByField,
+      orderByDirection
+    } = this.state;
+    return React.createElement(DynamicDataTable, _extends({
+      rows: rows,
+      currentPage: currentPage,
+      totalPages: totalPages,
+      orderByField: orderByField,
+      orderByDirection: orderByDirection,
+      changePage: this.changePage.bind(this),
+      changeOrder: this.changeOrder.bind(this)
```

Comment:
Unfortunately calling `.bind` in the render function still causes the issue as it creates a new function each time it is called causing the component to re-render unnecessarily. You need to refactor out the bind methods to the constructor.

Reply:
Moved binding to constructor.